### ISSUE TITLE
Adding guidance on read_csv() and error handling

### ISSUE THREAD

**Issue by jasonajones73** [datacarpentry/R-ecology-lesson#555]
As a lesson contribution, I was wondering how relevant it may be to add some guidance regarding using the `read_csv()` function to the [Starting with Data Episode](https://datacarpentry.org/R-ecology-lesson/02-starting-with-data.html). Specifically around handling parsing failures when you don't have a dataset that is as clean as the survey data used for this workshop. 

This issue came up during a live demo I was doing and I noticed that there is some back and forth in these lessons between `read.csv()` and `read_csv()` with no explanation about how to dig around in parsing failures and then handle those.

I have drafted some sample code to demonstrate what I am talking about and included it below.

```r
# Load tidyverse package
library(tidyverse)

# Download file from City of Raleigh Open Data portal
download.file(url="https://opendata.arcgis.com/datasets/24c0b37fa9bb4e16ba8bcaa7e806c615_0.csv",
              destfile = "police_data.csv")

# Read data in and assign to object "dat"
dat <- read_csv("police_data.csv")

# Notice Warning: 784165 parsing failures...See problems(...) for more details
# What can you do with this?

# Create an errors object
errors <- problems(dat)

# Look at errors neatly
View(errors)

# Print columns and failure counts to the console for easy correcting
table(errors$col)

# Fix your read_csv function call
dat <- read_csv("https://opendata.arcgis.com/datasets/24c0b37fa9bb4e16ba8bcaa7e806c615_0.csv",
                col_types = cols(city_of_incident = col_character(),
                                 latitude = col_number(),
                                 longitude = col_number(),
                                 reported_block_address = col_character()))
```

**Comment by anacost** [comment]
thank you, @jasonajones73, for your contribution!
This lesson should be updated with only one of these commands. Since the lesson will introduce tidyverse from the beginning, only read_csv should be presented in the future.

**Comment by fmichonneau** [comment]
addressed with #663
